### PR TITLE
Added workflow tests

### PR DESCRIPTION
**Pull request by nicholas-chiu**
Adding workflow tests based off of available workflow guides described in developers.lilt.com. Create Data Source and Data Source Management workflows have been merged into a single file as Create leads into Management cleanly.

### CODE REVIEW

File: workflow_tests/testCreateContent.js [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+      // AMD.
+      define(['expect.js', process.cwd()+'/src/index'], factory);
+    } else if (typeof module === 'object' && module.exports) {
+      // CommonJS-like environments that support module.exports, like Node.
+      factory(require('expect.js'), require(process.cwd()+'/src/index'));
+    } else {
+      // Browser globals (root is window)
+      factory(root.expect, root.LiltNode);
+    }
+}(this, function(expect, LiltNode) {
```
Comment (tgallant) [inline workflow_tests/testCreateContent.js:221]: why do we need to do this thing?

File: package.json [hunk 2]
```diff
@@ -7,13 +7,14 @@
   "scripts": {
     "build": "babel src -d dist",
     "prepack": "npm run build",
-    "test": "mocha --require @babel/register --recursive"
+    "test": "mocha ./workflow_tests/testInstantTranslation.js --require @babel/register --recursive --timeout 120000"
```
Comment (tgallant) [inline package.json:10]: should this point to the folder instead of this particular file?

**Review by tgallant — COMMENTED** [review]
looking good to me! just some random comments

File: workflow_tests/testCreateContent.js (hunk 1 above)
Comment (nicholas-chiu) [reply workflow_tests/testCreateContent.js:221]: it doesn't import LiltNode correctly without it and this is how the autogenerated test code imports the module, don't think it's all necessary but thought it was good to maintain the same pattern as the autogenerated tests

**Review by tgallant — APPROVED** [review]

**Pull request merged**